### DISCUSSION TITLE
2024 February 12 US Timezone Community Sync

### DISCUSSION THREAD

**Discussion by errose28** [apache/ozone#6210] [Community Sync Meetings]
## Attendees:
- Cloudera
- G-Research

### New and improved Grafana Dashboards

- @kerneltime 
- Existing dashboards have some regressions.
  - Metric names change, metrics are not updated accurately by Ozone.
- Ideas to improve dashboard stability:
  - Some type of "lock-file" for metrics. Similar to what we do to detect dependency changes.
  - Cross check metric names from JMX with those referenced in the Grafana json configurations.

### Improved client side read checksums using direct buffers

- @duongkame  @kerneltime
- See #6162 

### Developments on the new Ozone website

- @errose28 
- A draft CI pipeline exists on [this fork](https://github.com/errose28/ozone-site/actions/runs/7824638849). Planning to commit it in multiple smaller PRs to the ozone-site repo.
- First complete documentation page in the new website is done (thanks @adoroszlai). Check it out at https://ozone-site-v2.staged.apache.org/docs/developer-guide/project/release-guide to see how Docusaurus pages look and feel compared to Confluence or our current Hugo site.
- Social media share preview has been fixed on the new website as part of https://github.com/apache/ozone-site/pull/72. Try sharing a link to https://ozone-site-v2.staged.apache.org/ in slack or other social media sites to see the preview, and report formatting issues if you see any.

### Vote thread to change the main/default branch of the apache/ozone-site repo.

- @errose28 
- See [this vote thread](https://lists.apache.org/thread/1gk1jsnjh7y5wg0rzycgtr4xyq9noq1m)

### Open discussion on improving community sync structure

- @errose28 
- Add ideas to #6157

**Comment by kerneltime** [comment]
[Zoom recording](https://cloudera.zoom.us/rec/share/tXNBTxIgOc2OSQo8hJalGlqaW3wSAYXNB18KfSBGeMUc8dJuf1sJUzsfgOGUAb-6.FFKOi5IyDgSomG2K
)
Passcode: n9kc^7Qf
